### ISSUE TITLE
Documentation needed for system requirements

### ISSUE THREAD

**Issue by j-dornbusch** [samvera/hyrax#3470]
### Descriptive summary

To help new adopters to estimate hardware requirements, it would be useful to provide information on typical infrastructure sizing (RAM, disk and CPU) in Hyrax Management Guide.
An updated version of [this page](https://wiki.duraspace.org/display/samvera/Deployment+Hardware+Information).
Or like [this page from Vivo documentation](https://wiki.duraspace.org/display/VIVODOC19x/System+Requirements#SystemRequirements-HardwareRecommendations), but it is even more interesting to have real examples with the characteristics of the hardware, information about the architecture (workers servers for sidekiq? replicated storage? Solr scaling technics ? ) and a glimpse of the volumetry of data.

### Rationale

This will help people to anticipate infrastructure costs at the beginning of the project, when the behavior of the application is not very familiar yet.

**Comment by no-reply** [comment]
is this something we feel comfortable closing given CONTAINERS.md, docker-compose, and helm?

cc: @jeremyf, @orangewolf, @j-dornbusch

**Comment by jeremyf** [comment]
As someone that doesn't currently have this system running at my org/institution, I can't do anything to say "This is our realized experience."

However, I can say that the Containers, docker-compose, and helm provides repeatable documentation on what is needed to run the application.

tl;dr: Close it

**Comment by orangewolf** [comment]
we ran through these and they make sense to me, I say close it.

**Comment by j-dornbusch** [comment]
Thank you for your help.
We will then refer to the docker-compose and the Container.md files.
You may close this issue.

**Comment by rjkati** [comment]
@jlhardes seems like the consensus is to close this ticket since the proposed documentation is present elsewhere.
